README/README.md

```diff
@@ -0,0 +1,163 @@
+Installation
```

Comment:
i understand this is generated from Rmd. If yes, this should be top level README.md and needs to have 

> ## R client for kdb+
>
>Execute kdb+ queries from R for advanced high-performance analytics.
>
> ## See [Interfacing with R](http://code.kx.com/q/interfaces/with-r/) on Kx wiki.